### PR TITLE
Update Install Prerequisites script for macos

### PR DESCRIPTION
**Pull request by bugobliterator**
Also adds the test for the same in CI

### CODE REVIEW

**Comment by jmachuca77** [comment]
Can we merge the other PR first... there are more issues than just this.
#16667

**Comment by jmachuca77** [comment]
On second thought.... Im going to close the other one...
have you tested with Big Sur, and Monterrey

**Comment by bugobliterator** [comment]
> On second thought.... Im going to close the other one...
> have you tested with Big Sur, and Monterrey

@jmachuca77 Yes I tested it on Big Sur. CI checks on Catalina, I have not run it on Monterey. Monterey is too new, don't think we need to test on that yet.

File: .github/workflows/macos_build.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Prerequisites
+        shell: bash
+        run: |
+          Tools/environment_install/install-prereqs-mac.sh -y
+          source ~/.bash_profile
+          echo $PATH
+          ./waf configure --board CubeOrange
```
Comment (khancyr) [inline .github/workflows/macos_build.yml:20]: We should make a matrix and build all vehicles, that way we profit of a new check

Comment (bugobliterator) [reply .github/workflows/macos_build.yml:20]: @khancyr feel free to push any required changes to this workflow. I have already given you push access to my repo.

Comment (bugobliterator) [reply .github/workflows/macos_build.yml:20]: There is also ccache available on macos, we can probably use that as well.

**Comment by rmackay9** [comment]
@bugobliterator, in case you missed it there is some feedback to this PR here https://discuss.ardupilot.org/t/errors-setting-up-build-environment-on-mac/72882/7

**Comment by bugobliterator** [comment]
> @bugobliterator, in case you missed it there is some feedback to this PR here https://discuss.ardupilot.org/t/errors-setting-up-build-environment-on-mac/72882/7

Thanks I have responded.

**Review by khancyr — APPROVED** [review]
LGTM

**Review by tridge — APPROVED** [review]

**Pull request merged**